presto-native-execution/presto_cpp/main/TaskResource.cpp

```diff
@@ -112,14 +112,21 @@ void TaskResource::registerUris(http::HttpServer& server) {
       R"(/v1/task/async/(.+)/results/([0-9]+)/([0-9]+))",
       [&](proxygen::HTTPMessage* message,
           const std::vector<std::string>& pathMatch) {
-        return getResults(message, pathMatch);
+        return getResults(message, pathMatch, false);
```

Comment:
@arhimondr I'm seeing that Prestissimo supports both /task/async/ and task/ endpoints. Andrii, do we need to support async endpoint? I believe you have cleaned up Presto to remove that recently.

Reply:
Yes, the blocking implementation of that endpoint has been removed and now `/task/<>/results` points to the non blocking implementation. The `/task/async/` endpoint is no longer available.

Reply:
@arhimondr Andrii, thank you for clarifying.

@Yuhta Jimmy, let's remove /task/async endpoint from Prestissimo as well.